data.sql

```diff
@@ -1,5 +1,12 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23, 'Unknown'),
+       ('Gabumon', '2018-11-15', 2, TRUE, 8, 'Unknown'),
+       ('Pikachu', '2021-01-07', 1, FALSE, 15.04, 'Unknown'),
+       ('Devimon', '2017-05-12', 5, TRUE, 11, 'Unknown'),
+       ('Squirtle', '1993-04-02', 3, FALSE, -12.13, 'Unknown'),
+       ('Angemon', '2005-06-12', 1, TRUE, -45, 'Unknown'),
+       ('Boarmon', '2000-06-07', 7, TRUE, 20.4, 'Unknown'),
```

Comment:
- [ ] @DevKaranJ Kindly check the date of birth for `Boarmon` you have entered the incorrect date.